lib/didww/resource/nanpa_prefix.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module DIDWW
+  module Resource
+    class NanpaPrefix < Base
+      has_one :country, class_name: Country
+      has_one :region, class_name: Region
+
+      property :npa, type: :string
+      # Type: String
+      # Description: NPA codes, or area codes consisting of three digits
+      property :nxx, type: :string
+      # Type: String
+      # Description: Refer to three-digit code
+      # that forms the second part of a 10-digit North American phone number
```

Comment:
why are comments behind the code?

Reply:
because this style is used for entire gem